v2/internal/testcommon/resource_namer.go

```diff
@@ -103,6 +105,18 @@ func (n ResourceNamer) makeRandomStringOfLength(num int, runes []rune) string {
 	return string(result)
 }
 
+func (n ResourceNamer) makeSecureStringOfLength(num int, runes []rune) string {
```

Comment:
Before, these passwords were deterministic, and now they are not.

Should we seed the crypto randomizer? 
I guess there's no way to do that in the crypto API?

Might be worth making a note on this method and maybe the password method too that it's not stable...